ipt_connect/IPTdev/model_SupplementaryMaterial.py

```diff
@@ -1,16 +1,12 @@
 # coding: utf8
 
-import models as ipt_connect_models
-
 from django.db import models
 
-import parameters as params
```

Comment:
Despite the fact that the file has not been used, we should better import it - just because it is useful to suppose `params` to be available everywhere in the code.

Singleton pattern as is =/